db/migrate/20200108230110_create_teams.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTeams < ActiveRecord::Migration[5.2]
```

Comment:
This migration was previously created in #2. We should probably sync about this.

Reply:
Honestly, I don't agree with adding a table per PR, especially for the foundational data model, because the existence of the tables depends on defining how they relate with each other.

Anyway, I suggest that we don't close #2, merge this first, rebase & force push on the branch for that PR so we can incorporate any important additions?